collector_test.go

```
@@ -0,0 +1,941 @@
+	return nil, e.err
+}
+
+type fakeSTS struct {
+	account string
+}
+
+func (f *fakeSTS) GetCallerIdentity(context.Context, *sts.GetCallerIdentityInput, ...func(*sts.Options)) (*sts.GetCallerIdentityOutput, error) {
+	return &sts.GetCallerIdentityOutput{Account: aws.String(f.account)}, nil
+}
+
+type recordingSTS struct {
```

golangci-lint (2.12.2)

[error] type recordingSTS is unused

(unused)